lib/config-loader.js

```diff
@@ -135,15 +135,18 @@ module.exports = () => {
   // 4. Load manifest config
   if (config.app.hasBackend) {
     config.manifest.packagePlaceholder = '__APP_PACKAGE__'
+    // TODO: migrate existing projects that have the packagePlaceholder in their manifest.yml
```

Comment:
Thinking out loud, should we make the effort to be backward compatible?
- in that case we could still support the placeholder and rename it here to pkg-version + warn with a deprecation notice
or we provide a tool to migrate the manifest config

Reply:
(Personally I'd prefer the support for the placeholder + deprecation notice. Gives more time to update).